### DISCUSSION TITLE
How can I make text align and font-size ?

### DISCUSSION THREAD

**Discussion by zoom2009** [yoopta-editor/Yoopta-Editor#180] [Q&A] [answered]
I was tried using `@yoopta/marks` but It's seem support only `Bold, Italic, CodeMark, Underline, Strike, Highlight`.
Please suggest me, How Can I make custom style or dropdown for selection font-size and text align.

thank you so much, your lib is amazing

**Comment by zoom2009** [comment]
I found 1 solution using custom mark. Select text and press `command + p` this will center.
but hope that will be good If I can make custom plugin, I think that in-progress.
thank you

```typescript
const CUSTOM_CENTER_MARK = createYooptaMark<SuperscriptMarkProps>({
  type: 'center',
  render: (props) => {
    return <div className="text-center">{props.children}</div>;
  },
  hotkey: 'mod+p'
})
```

**Comment by Darginec05** [comment, marked as answer]
Hey @zoom2009 
Just added alignment support for blocks. 

It's available in version `v4.6.2`. 
Now you can change align using toolbar.
For void blocks (Image, Video, File, Embed) it's available in dots options

Check out it in example: https://yoopta.dev/examples/withBaseFullSetup

**Reply by zoom2009** [reply]
thank alot 👍
